spyderlib/plugins/inspector.py

```diff
@@ -199,6 +199,42 @@ def setup_page(self):
         self.setLayout(vlayout)
 
 
+class FrameWebView(QFrame):

```

Comment:
Please move this widget to

    spyderlib/widgets/browser.py

because that's where we put all `QWebKit` widgets :-)

This is the only comment I have about this before merging (other than good job!)